### PR TITLE
WIP: skip pytest_ignore_collect with -k/-m

### PR DESCRIPTION
**Pull request by blueyed**
Ref: https://github.com/tarpas/pytest-testmon/issues/135

### CODE REVIEW

File: testmon/pytest_testmon.py [hunk 1]
```diff
@@ -150,6 +150,10 @@ def __init__(self, config, testmon_data):
         self.f_to_ignore = self.testmon_data.stable_files
         if self.config.getoption('tlf'):
             self.f_to_ignore -= self.testmon_data.f_last_failed
+        self.has_possibly_filtering_args = any(
+            x for x in config._origargs
```
Comment (blueyed) [inline testmon/pytest_testmon.py]: Not a public attribute: https://github.com/pytest-dev/pytest/issues/5106

**Comment by tarpas** [comment]
I think think this would be a change for the better. I quite dislike the name has_possibly_filtering_args :)

File: testmon/pytest_testmon.py (hunk 1 above)
Comment (tarpas) [reply testmon/pytest_testmon.py]: how about has_k_m_args , or has_filtering_args?

Comment (blueyed) [reply testmon/pytest_testmon.py]: `has_filtering_args` would be ok, but it is not given that they actually filter anythings - that's why I had the longer name I guess.

**Comment by blueyed** [comment]
Not anytime soon.  I also think that other PRs are more important first.
I.e. would be nice to get #106 sorted out, and the other one for the config.

**Comment by tarpas** [comment]
Thanks, this class of reports inspired me to rewrite testmon, so this is hopefully not relevant any more.

**Pull request closed without merging**